v23.1/architecture/replication-layer.md

```diff
@@ -207,6 +209,20 @@ Rebalancing is achieved by using a snapshot of a replica from the leaseholder, a
 
 In addition to the rebalancing that occurs when nodes join or leave a cluster, replicas are also rebalanced automatically based on the relative load across the nodes within a cluster. For more information, see the `kv.allocator.load_based_rebalancing` and `kv.allocator.qps_rebalance_threshold` [cluster settings](../cluster-settings.html). Note that depending on the needs of your deployment, you can exercise additional control over the location of leases and replicas by [configuring replication zones](../configure-replication-zones.html).
 
+### Important values and timeouts
+
+The following table lists some important values used by CockroachDB's replication layer:
+
+Constant | Default value | Notes
+---------|---------------|------
+[Raft](#raft) election timeout | {{site.data.constants.raft_election_timeout}} | The actual election timeout is multiplied by a random factor to avoid ties.
+[Raft](#raft) proposal timeout | {{site.data.constants.raft_proposal_timeout}} |
+[Node heartbeat interval](#how-leases-are-transferred-from-a-dead-node) | {{site.data.constants.node_heartbeat_interval}} |
+[Lease interval](#how-leases-are-transferred-from-a-dead-node) | {{site.data.constants.lease_interval}} |
+[Lease acquisition timeout](#how-leases-are-transferred-from-a-dead-node) | {{site.data.constants.lease_acquisition_timeout}} |
+
+The values of the intervals above can be tuned using the environment variable `COCKROACH_RAFT_ELECTION_TIMEOUT_TICKS`, which specifies the minimum number of Raft ticks must pass before holding an election. It defaults to {{site.data.cockroach_raft_election_timeout_ticks}}. If changed, it will scale these bintervals proportionally.
```

Comment:
This is no longer entirely true. In cockroachdb/cockroach#98616 we split out the lease interval to a separate env var `COCKROACH_RANGE_LEASE_DURATION` with a default of 6 seconds, which is independent of the Raft election timeout -- the node heartbeat interval is half of this, so 3 seconds. Similarly, cockroachdb/cockroach#98559 split out the proposal timeout to `COCKROACH_RAFT_REPROPOSAL_TIMEOUT_TICKS` defaulting to 6 ticks (3 seconds given the current 500 ms tick interval).

Reply:
I've revised this table to explain that these new environment variables can be used, as well as updated the table so that the various values are shown as what they are calculated from, e.g. "n_ticks * tick_value" (where appropriate)

also updated the table to use the updated tick info (and env var) from your other comment